README.md

```diff
@@ -1,2 +1,66 @@
 # Pancake
-Pancake is a high-performance encryption proxy for key-value stores that uses frequency-smoothing to prevent damaging access pattern attacks.
\ No newline at end of file
+
+`Pancake` is the first system designed to protect key-value stores from access pattern leakage attacks with small constant factor bandwidth overhead.
```

Comment:
Maybe add the second sentence about frequency smoothing as well (from the abstract)?